### PR TITLE
Use different emoji for project create

### LINKED ISSUE #17876: Quarkus CLI does not display icons correctly
## Describe the bug
When creating an app via Quarkus CLI the icons are not displayed as shown in the documentation.
In my terminal a white rectangle is shown instead.

### Expected behavior
Icons are displayed correctly or if the terminal does not support this something else is used instead.

### Actual behavior
Icons are not displayed correctly.

## To Reproduce

1. quarkus create app

### Screenshots
quarkus-cli-missing-logos

## Environment
OS: Linux
Shell: Bash
Terminal: Konsole (KDE, https://konsole.kde.org/)

### PR DESCRIPTION
**Pull request by ebullient**
Resolves #17876

### CODE REVIEW

**Review by maxandersen — APPROVED** [review]
lgtm

**Comment by axblu** [comment]
@ebullient Just as a reminder: do not forget to also update the CLI guide with the new icons.

**Comment by ebullient** [comment]
> @ebullient Just as a reminder: do not forget to also update the CLI guide with the new icons.

Done. Including a note that the emoji may not match exactly, and may not (in fact) be rendered at all. IntelliJ has some long-running issues re: correct rendering of emoji in the console. (e.g. my terminal in linux is fine, terminal and intellij on mac is fine, terminal in intellij on linux is not.. and there are open issues for intellij about emoji in the terminal, so I'm not inclined to fuss overmuch about it)

File: docs/src/main/asciidoc/cli-tooling.adoc [hunk 1]
```diff
@@ -89,23 +89,24 @@ $ quarkus create
...
-🛠 dockerfiles
-🛠 maven-wrapper
-🐒 resteasy-codestart
+📚  java
+🔨  maven
+📦  quarkus
+📝  config-properties
+🔧  dockerfiles
+🔧  maven-wrapper
+📄  resteasy-codestart
```
Comment (gastaldi) [inline docs/src/main/asciidoc/cli-tooling.adoc:98]: Darn, I liked the little monkey

**Pull request merged**